docs/data-sources/modelarts_datasets.md

```diff
@@ -0,0 +1,110 @@
+---
+subcategory: "Beta"
```

Comment:
Use ModelArgs subcategory instead of Beta, and move other resources/data sources out from Beta as well.